app/config/local.php.ses

```diff
@@ -0,0 +1,782 @@
+<?php
```

Comment:
This was probably pushed by accident, right? Please remove it and check if you haven't published some secrets. If so, reset those just to be sure.

Reply:
Yes, this was by accident. Thank you